src/main/java/com/sourcegraph/cody/CodyToolWindowFactory.java

```diff
@@ -38,22 +34,10 @@ public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindo
           customCodySettings.add(new OpenCodySettingsEditorAction());
           customCodySettings.addSeparator();
           toolWindow.setAdditionalGearActions(customCodySettings);
-          List<AnAction> titleActions = new ArrayList<>();
-          createTitleActions(titleActions);
-          if (!titleActions.isEmpty()) {
-            toolWindow.setTitleActions(titleActions);
-          }
           return null;
         });
   }
 
-  private void createTitleActions(@NotNull List<? super AnAction> titleActions) {
-    AnAction action = ActionManager.getInstance().getAction("cody.newChat");
```

Comment:
The action remains in general - it's just removed from the tool window. So we will be able to implement a new behaviour - a shortcut action opening the chat. Is tthat right?